modules/apps/knowledge-base/knowledge-base-api/src/main/java/com/liferay/knowledge/base/service/KBArticleServiceWrapper.java

```diff
@@ -186,12 +186,13 @@ public int getGroupKBArticlesCount(long groupId, int status) {
 
 	@Override
 	public String getGroupKBArticlesRSS(
-			int status, int rssDelta, String rssDisplayStyle, String rssFormat,
-			com.liferay.portal.kernel.theme.ThemeDisplay themeDisplay)
+			String displayStyle, int max, int status,
+			com.liferay.portal.kernel.theme.ThemeDisplay themeDisplay,
+			String type, double version)
```

Comment:
What criteria did you use to sort the parameters? If you look for usages of `ThemeDisplay` in services, it is always the last parameter.

Reply:
Sorting policy: alphabetically. Is there another rule I should know? If I put `ThemeDisplay` at the end, what about the rest of parameters?

Reply:
The alphabetical rule applies 100% for private stuff. Public APIs have a complex set of rules. In general, the sorting should be something similar to this: companyId > groupId > entryId > ... > ThemeDisplay >= ServiceContext. That it, first scope identifiers (instance and group ids), the entry identifiers, then miscelaneous fields and finally service context and theme display.

You'll have to make a guess about the miscellaneous parameters. Alphabetical order may be fine for these.

Reply:
What do you think about using this order? `long resourcePrimKey, int status, int max, String type, double version, String displayStyle, ThemeDisplay themeDisplay`

This is similar to the order found in other Services like this:
`BlogsEntryService.getCompanyEntriesRSS(
			long companyId, Date displayDate, int status, int max, String type,
			double version, String displayStyle, String feedURL,
			String entryURL, ThemeDisplay themeDisplay)`

Reply:
Sounds good.

Reply:
Done